mlflow/R/mlflow/DESCRIPTION

```diff
@@ -1,14 +1,14 @@
 Package: mlflow
 Type: Package
 Title: Interface to MLflow
-Version: 0.1.0
+Version: 0.7.0
 Authors@R: c(
     person("Javier", "Luraschi", email = "javier@rstudio.com", role = c("aut", "cre")),
     person("Kevin", "Kuo", role = c("aut"), email = "kevin.kuo@rstudio.com",
            comment = c(ORCID = "0000-0001-7803-7901")),
     person(family = "RStudio", role = c("cph"))
     )
-Maintainer: The package maintainer <yourself@somewhere.net>
+Maintainer: Javier Luraschi <javier@rstudio.com>
```

Comment:
super nit: this line is redundant :wink:

Reply:
I vaguely remember some parsers in RStudio require the maintainer field to be defined explicitly since `Maintainer` is an old filed while `Author@R` is the new one, but I might be wrong.